src/components/Tooltip/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface TooltipProps {
+    /**
+	 * The Tooltip ID will be generated if not assigned
+	 */
+	id?: string;
+
+	/**
+	 * The text in Tooltip
+	 */
+	text?: string;
```

Comment:
I don't think this should be optional, if the dev wants a tooltip they should be prompted to add text